examples/00-EDB/09_Configuration.py

```diff
@@ -73,61 +70,69 @@
 #         }
 #     }
 # }
+# ```
+#
+# The method ``Edb.components.import_definitions`` imports the componet definitions that map electrical models to the components in the simulaton model.
 
-edbapp.components.import_definition(os.path.join(target_aedb, "1_comp_definition.json"))
+edbapp.components.import_definition(os.path.join(target_aedb, 
+                                                 "1_comp_definition.json"))
 
 ###############################################################################
-# Import BOM
-# ~~~~~~~~~~
-# This step imports a BOM file in CSV format. The BOM contains the
-# reference designator, part name, component type, and default value.
-# Components not in the BOM are deactivated.
-# In this example the csv file is stored for convenience in aedb folder.
+# ## Import BOM
 #
+# The bill of materials (BOM) file provides the list of all components
+# by reference designator, part name, component type, and nominal value.
+#
+# Components that are not contained in the BOM are deactivated in the
+# simulation model.
+# In this example the csv file was saved in the aedb folder.
+#
+# ```
 # +------------+-----------------------+-----------+------------+
 # | RefDes     | Part name             | Type      | Value      |
 # +============+=======================+===========+============+
 # | C380       | CAPC1005X55X25LL05T10 | Capacitor | 11nF       |
 # +------------+-----------------------+-----------+------------+
-
+# ```
+#
+# Having red the informaton in the BOM and definitions file, electrical models can be assigned to all of the components in the simulation model.
 
 edbapp.components.import_bom(os.path.join(target_aedb, "0_bom.csv"),
                                   refdes_col=0,
                                   part_name_col=1,
                                   comp_type_col=2,
                                   value_col=3)
 
-
-
 ###############################################################################
-# Check Component Values
-# ~~~~~~~~~~~~~~~~~~~~~~
+# ## Veriify a Component
+#
 # Component property allows to access all components instances and their property with getters and setters.
 
 comp = edbapp.components["C1"]
 comp.model_type, comp.value
 
-
 ###############################################################################
-# Check Component Definition
-# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+# ## Check Component Definition
+#
 # When an s-parameter model is associated to a component it will be available in nport_comp_definition property.
```

Comment:
```suggestion
# When an s-parameter model is associated with a component, it is available in the ``nport_comp_definition`` property.
```
Determine if it should be "s-parameter" or "S-parameter" and make all occurrences consistent.